apps/authorization/models.py

```diff
@@ -70,11 +73,217 @@ def update_grants(*args, **kwargs):
 
 def check_grants():
     AccessToken = get_access_token_model()
-    token_count = AccessToken.objects.filter(
-        expires__gt=timezone.now(),
-    ).values('user', 'application').distinct().count()
+    token_count = (
+        AccessToken.objects.filter(
+            expires__gt=timezone.now(),
+        )
+        .values("user", "application")
+        .distinct()
+        .count()
+    )
     grant_count = DataAccessGrant.objects.all().count()
     return {
         "unique_tokens": token_count,
         "grants": grant_count,
     }
+
+
+def get_grant_counts():
+    # Grant real/synth bene counts (includes granted to multiple apps)
+    start_time = datetime.utcnow().timestamp()
+
+    real_bene_cnt = DataAccessGrant.objects.filter(
+        ~Q(beneficiary__crosswalk___fhir_id__startswith="-")
+        & ~Q(beneficiary__crosswalk___fhir_id="")
+    ).count()
+
+    synth_bene_cnt = DataAccessGrant.objects.filter(
+        Q(beneficiary__crosswalk___fhir_id__startswith="-")
+        & ~Q(beneficiary__crosswalk___fhir_id="")
+    ).count()
+
+    bene_cnt_elapsed = round(datetime.utcnow().timestamp() - start_time, 3)
+
+    # Grant real/synth bene distinct counts (excludes granted to multiple apps)
+    start_time = datetime.utcnow().timestamp()
+
+    real_bene_distinct_cnt = (
+        DataAccessGrant.objects.order_by("beneficiary__username")
```

Comment:
Have you done any perf testing on these queries?

Reply:
@nbragdon Not yet. Planning to start testing this branch in TEST, SANDBOX, PROD via canaries soon (today).

I've been reviewing metrics in NewRelic for the 3AM time frame when this nightly job is currently running. It doesn't impact the system very much per the Postgres related metrics.  

Also, the Jenkins job has been taking about 2-minutes to complete, but this includes the container setup and other stuff before the Django command is run.

This PR includes timings for the queries, so we can get more information. Currently we can't tell from the Jenkins logs.

Will report findings from testing to review before we can approve & merge this in.